openmetadata-ui/src/main/resources/ui/src/utils/FeedElementUtils.tsx

```diff
@@ -33,13 +35,28 @@ export const getFieldThreadElement = (
 
   return !isEmpty(threadValue) ? (
     <p
-      className="tw-text-right link-text"
+      className="tw-text-right link-text tw-text-base"
       onClick={(e) => {
         e.preventDefault();
         e.stopPropagation();
         onThreadLinkSelect?.(threadValue.entityLink);
       }}>
-      <i className="far fa-comment" /> {threadValue.count} threads
+      <SVGIcons alt="comments" icon={Icons.COMMENT} width="18px" />{' '}
+      {threadValue.count}
     </p>
   ) : null;
 };
+
+export const getDefaultValue = (owner: EntityReference) => {
+  if (isUndefined(owner)) {
+    return;
```

Comment:
If the owner is not defined, can we have the message without the mention? The users will then have an option to tag a user with the message.